src/Core/Manifest/ClassManifest.php

```diff
@@ -516,6 +644,12 @@ protected function coalesceDescendants($class)
      */
     protected function validateItemCache($data)
     {
+        if (!$data) {
+            return false;
+        }
+        if (is_array($data)) {
```

Comment:
shouldn't this be `if (!is_array($data)) {`? at least that's what it looks like from the code below